src/_guides/language/effective-dart/design.md

```diff
@@ -743,6 +743,24 @@ Future<bool> install(PackageId id, String destination) {
 
 With types, all of this is clarified.
 
+
+### DON'T specify that a setter returns `void`.
```

Comment:
This reads to me like it might be discouraging settings *in general*. How about:

```markdown
### DON'T specify a return type for a setter.
```